beacon-chain/state/stategen/getter_test.go

```diff
@@ -139,11 +139,14 @@ func TestStateByRootInitialSync_UseCache(t *testing.T) {
 	require.NoError(t, service.beaconDB.SaveStateSummary(ctx, &pb.StateSummary{Root: r[:]}))
 	service.hotStateCache.Put(r, beaconState)
 
-	loadedState, err := service.StateByRoot(ctx, r)
+	loadedState, err := service.StateByRootInitialSync(ctx, r)
```

Comment:
Note: this test was called `TestStateByRootInitialSync_UseCache` but was calling a different method. Oops!